### ISSUE TITLE
Bad Uploads, Google Drive and Zenodo Software

### ISSUE THREAD

**Issue by sfisher** [CDL-Dryad/dryad-product-roadmap#1122]
It will fail to replicate to Zenodo (understandably) when the user puts in a bad google drive URL.

A URL like https://drive.google.com/drive/folders/1eaJyz0xrhcKJGortJIYUUNgXpo3B-grA?usp=sharing is a link to a web page.  This isn't going to upload all the things listed in the web page.

It was unable to fill in a file name for it when it validated in the UI and showed status code 409 from Google.

It then gave a very weird error of "You should never start replicating unless starting from an enqueued state" even though it seemed like it began replicating.

This check happens at the first before replication, but it seemed to try to replicate before this happened.  I suspect it had a different error in replication and then was retried again and then overwrote the original error with this one.

When I gave this dumb google share a filename in the database and tried replication again then it succeeded, even though this is garbage-y data for a user to put in.

**Comment by sfisher** [comment]
I think this particular thing was a one-off so closing.  We still have some drive/dropbox issues which I think have been very low priority to fix, though.
